config/config.go

```diff
@@ -0,0 +1,30 @@
+package config
```

Comment:
This is not wrong.
But, the idea of having a config file, was to use a yml or json file with all the values you will need. And, in the config.go file, read that file, and create a struct in memory with those values, to be used in the main file.